inlong-sort/sort-connectors/hbase/src/main/java/org/apache/inlong/sort/hbase/sink/HBaseSinkFunction.java

```diff
@@ -206,7 +205,6 @@ public void open(Configuration parameters) throws Exception {
             LOGGER.error("Exception while creating connection to HBase.", ioe);
             throw new RuntimeException("Cannot create connection to HBase.", ioe);
         }
-        LOGGER.info("end open.");
```

Comment:
I suggest save this log, It can help analysist problem

Reply:
done